.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ executors:
 parameters:
     current_golden_images_hash:
         type: string
-        default: aa8a18f56f8af4fe5526f060006ad0c810699c73
+        default: b268071af940937c61fd0b5e458a69f825c68a48
```

Comment:
Interesting - can you tell me about this hash value?

Reply:
The goal is for this to be explained here: https://github.com/adobe/spectrum-web-components#screenshot-testing and we can talk about it some more in person later. You and I have also talked separately about alternative approaches to this concept that I'd be very happy to see explored in due time.